Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1295,7 +1295,7 @@ theorem MulAction.isClosedMap_quotient [CompactSpace α] :
     IsClosedMap (Quotient.mk' : β → Quotient (orbitRel α β)) := by
   intro t ht
   rw [← quotientMap_quotient_mk'.isClosed_preimage, MulAction.quotient_preimage_image_eq_union_mul]
-  convert ht.smul_left_of_isCompact (isCompact_univ (α := α))
+  convert ht.smul_left_of_isCompact (isCompact_univ (X := α))
```

Comment:
This is possibly evidence that the `α` name was better

Reply:
I agree this demonstrates an inconsistency. My explanation is that the section on line 1260 onwards uses Greek letters. (I'd be happy to change that also. I decided to start with one file.)

Reply:
For zulip, it seems going ahead with this is wanted. Shall I change this section in this PR also?